packages/angular/projects/clr-angular/src/data/datagrid/datagrid.spec.ts

```diff
@@ -673,6 +718,43 @@ export default function (): void {
         const rows = this.context.clarityElement.querySelectorAll('.datagrid-cell');
         expect(['1', '1', '2', '4', '3', '9']).toEqual([...rows].map(r => r.textContent));
       });
+
+      it('sorting by column works when pagination is also added', function () {
+        // Instantiate the component only to obtain the test data inside
+        const componentInstance = new SortPaginationTest();
+        const testUsers = componentInstance.users;
+        const sortedTestUsers = [...testUsers].sort((a, b) => a.name.localeCompare(b.name));
+        const reverseSortedTestUsers = [...sortedTestUsers].reverse();
+
+        this.context = this.create(ClrDatagrid, SortPaginationTest);
+        const rows = this.context.clarityElement.querySelectorAll('.datagrid-cell');
+        expect(rows.length).toEqual(10); // 10 items * 1 property = 10 rows
+
+        rows.forEach((row, index) => {
+          expect(row.textContent).toEqual(testUsers[index].name);
+        });
+
+        const header = this.context.clarityElement.querySelector('.datagrid-column-title');
+        expect(header.textContent).toEqual('Name');
```

Comment:
I don't get why we expect to be something else than `Name` ?:) Is there something that I'm missing ?